### DISCUSSION TITLE
az login automation?

### DISCUSSION THREAD

**Discussion by dkirrane** [devcontainers/community#179] [Q&A]
What's the best way to handle credentials for az cli?
Do I have to manually run these commands each time I start a new devcontainer?
 `az login`, `az acr login`, `az account set --subscription XXX`

```
		"ghcr.io/devcontainers/features/azure-cli:1": {
			"version": "latest"
		}
```

**Comment by chrmarti** [comment]
Have you tried mounting the local `~/.azure` folder inside the container?
